pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -153,6 +153,10 @@ func Convert_v1_Volume_To_api_Disk(source *v1.Volume, disk *Disk, c *ConverterCo
 		return Convert_v1_HostDisk_To_api_Disk(source.HostDisk.Path, disk, c)
 	}
 
+	if source.PersistentVolumeClaim != nil {
```

Comment:
Shouldn't we use HostDisk as a vehicle between virt-handler and virt-launcher? 
@slintes look at https://github.com/kubevirt/kubevirt/pull/1349#discussion_r205722177
@rmohr do you agree?

I think with the current state of this PR we still have an issue https://github.com/kubevirt/kubevirt/issues/991

Reply:
@mfranczy in this case we are fine. We don't need a size or something like that. The block device is ready to use.

Reply:
@slintes forget about the comment above.